source/diagrams/manage-diagrams/rename-diagrams.txt

```diff
@@ -1,13 +1,62 @@
 .. _rm-rename-diagrams:
 
-===============
-Rename Diagrams
-===============
+================
+Rename a Diagram
+================
 
 .. contents:: On this page
-   :local:
-   :backlinks: none
-   :depth: 1
+   :local: 
+   :backlinks: none 
+   :depth: 1 
    :class: singlecol
 
-Rename diagrams.
\ No newline at end of file
+You can rename diagrams on the :guilabel:`Mapping` screen of Relational
+Migrator. Relational Migrator displays diagram names on the Diagram
+Toolbar. 
+
+About this Task 
+---------------
+
+- You can rename diagrams to customize your diagram properties. 
+- Relational Migrator displays your diagram names on the Diagram Toolbar
+  of the :guilabel:`Mapping` screen and in the :guilabel:`Diagram
+  Properties` pane. 
+
+Steps 
+-----
+
+.. procedure::
+   :style:  connected
+
+   .. step:: Navigate to the diagram you want to rename
+
+      In the Diagram Toolbar, click the name of the diagram  to display the
+      diagram. 
+
+      .. image:: /img/diagrams/diagram-toolbar.png
+         :alt: Diagram toolbar    
+   
+   .. step:: Open the :guilabel:`Diagram Properties` pane
+
+      Click the :icon-fa5:`chevron-circle-right` arrow button to open
+      the :guilabel:`Diagram Properties` pane. 
+
+      .. image:: /img/diagrams/diagram-properties.png
+         :alt: Diagram Properties pane
+      
+      The :guilabel:`Diagram Properties` pane displays the active
+      diagram name and notes. By default, Relational Migrator names
+      diagrams in the order they were created and leaves the notes field
+      empty. 
+
+   .. step:: Enter your new diagram name in the Name box
+
+      Relational Migrator automatically saves your new diagram name and
```

Comment:
A suggestion: "Relational Migrator automatically saves your new diagram name **after clicking away from the input**". If that is too specific, then ignore :)